src/views/CronTabList/CronTabRowActions.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { CronTabKind } from 'src/types';
+import {
+  Dropdown,
+  DropdownPosition,
+  KebabToggle,
+} from '@patternfly/react-core';
+
+type CronTabRowActionsProps = {
+  obj?: CronTabKind;
+};
+
+const CronTabRowActions: React.FC<CronTabRowActionsProps> = ({ obj }) => {
+  const [isDropdownOpen, setIsDropdownOpen] = React.useState(false);
+
+  return (
+    <Dropdown
+      menuAppendTo={"parent"}
+      onSelect={() => setIsDropdownOpen(false)}
+      toggle={<KebabToggle onToggle={setIsDropdownOpen} id="toggle-id-disk" />}
+      isOpen={isDropdownOpen}
+      isPlain
+      dropdownItems={[]}
```

Comment:
If there are no items, I would leave the dropdown out of the PR for now.

Reply:
wanted to preserve, if it would be used as a starting point for a new plugin, but can remove